vulcanai2/plotters/visualization.py

```diff
@@ -14,6 +14,8 @@
 
 from datetime import datetime
 
+from .utils import GuidedBackprop
+
```

Comment:
I don't see the `utils.py`, are you planning to add it as `utils/plotters.py`?

Reply:
yeah I plan on doing the folder restructure after I'm done